.github/workflows/on-pullrequest.yml

```diff
@@ -208,6 +207,24 @@ jobs:
           sudo systemctl start postgresql.service
           sudo -u postgres psql -U postgres -c "alter user postgres with password '${{env.POSTGRES_PASSWORD}}';"
 
+      - name: Create .env file
```

Comment:
I think that with this setup, we could stop requiring the e2e.env file in line #305 and move this to a common place in the workflow. I haven't reviewed both files line by line to see if it covers all required values